README.md

```diff
@@ -11,15 +11,85 @@ The KumaScript service takes requests to render raw documents (documents that
 may contain one or more embedded macro calls), and responds with a
 fully-rendered document (a document where each of the embedded macros
 has been executed and replaced inline with its output). The
-requests can be either via GET, with the full document path in the URL like
-`GET /docs/en-US/JavaScript/Foo`, or via `POST /docs`, with the actual raw
-content of the document included in the body of the POST. Here's an overview
-of a GET request:
+requests are via the `POST /docs` endpoint, with the raw
+content of the document to be rendered included in the body of the POST.
+
+### Source code
```

Comment:
I like this section, it is a good summary of your exploration and rewriting of the server code. However, I expect 99% of the changes will be to macros, so I think it should follow the section on updating macros.

Reply:
I ended up moving to to the end of the README instead.